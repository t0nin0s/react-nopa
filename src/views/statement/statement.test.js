import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Statement from './statement'

test('Statement component renders as expected', () => {
  const component = shallow(<Statement />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
