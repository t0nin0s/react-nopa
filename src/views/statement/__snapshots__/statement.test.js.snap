// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Statement component renders as expected 1`] = `
<Layout>
  <div
    className="statement"
  >
    <div
      className="content"
    >
      <section
        className="section section--primary"
      >
        <div
          className="container centered"
        >
          <h1
            className="desktopOnly section__h1"
          >
            Which bank does this account belong to?
          </h1>
          <p
            className="desktopOnly section__p"
          >
            Track of all your payments by connecting as
                          many bank accounts as you'd like to your
                          NOPA account and get updates your balance
                          instantly. Plus it's free.
          </p>
          <AccountDetail />
          <TransactionsTable
            transactions={
              Array [
                Object {
                  "amount": "35.98",
                  "date": "Today",
                  "description": "Tesco",
                  "type": "OUT",
                },
                Object {
                  "amount": "35.98",
                  "date": "Today",
                  "description": "Tesco",
                  "type": "OUT",
                },
                Object {
                  "amount": "35.98",
                  "date": "Today",
                  "description": "Tesco",
                  "type": "OUT",
                },
                Object {
                  "amount": "35.98",
                  "date": "22 November 2016",
                  "description": "Tesco",
                  "type": "OUT",
                },
                Object {
                  "amount": "35.98",
                  "date": "22 November 2016",
                  "description": "Tesco",
                  "type": "OUT",
                },
                Object {
                  "amount": "35.98",
                  "date": "21 November 2016",
                  "description": "Tesco",
                  "type": "OUT",
                },
                Object {
                  "amount": "35.98",
                  "date": "21 November 2016",
                  "description": "Tesco",
                  "type": "OUT",
                },
                Object {
                  "amount": "35.98",
                  "date": "20 November 2016",
                  "description": "Tesco",
                  "type": "OUT",
                },
              ]
            }
          />
          <a
            className="button button"
            href="#"
          >
            Show More
          </a>
        </div>
      </section>
    </div>
  </div>
</Layout>
`;
